### PR TITLE
Documented how to apply taints to manage mixed OS clusters in RKE2 

### LINKED ISSUE #3451: [Rancher2] Document how to apply taints to manage mixed OS clusters in RKE2
**Request Summary:**

Need to document how to apply taints in mixed OS clusters, see https://github.com/rancher/dashboard/issues/3921

As part of https://github.com/rancher/windows/issues/4

### PR DESCRIPTION
**Pull request by jtravee**
Closes #3451

### CODE REVIEW

File: content/rancher/v2.6/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ The RKE2 provisioning tech preview also includes installing RKE2 on Windows clus
 
 Windows Support for RKE2 Custom Clusters requires choosing Calico as the CNI.
 
+>**Important:** Rancher will allow Windows workload pods to deploy on both Windows and Linux worker nodes by default. When creating mixed clusters in RKE2, you must taint the Linux node to ensure Windows workloads will only be deployed to Windows nodes. Refer [here]({{<baseurl>}}/rancher/v2.6/en/logging/taints-tolerations/) for more information on taints and tolerations in Rancher.
```
Comment (btat) [inline content/rancher/v2.6/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md:46]: On https://github.com/rancher/dashboard/issues/3921#issuecomment-905642221 it mentions the dashboard messaging was `Updated to reflect move away from taints/tolerations and instead promote NodeSelector usage` so this should reflect that too.

**Review by btat — CHANGES_REQUESTED** [review]

File: content/rancher/v2.6/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md (hunk 1 above)
Comment (jtravee) [reply content/rancher/v2.6/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md]: Updated per feedback.

File: content/rancher/v2.6/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md [hunk 2]
```diff
@@ -43,6 +43,8 @@ The RKE2 provisioning tech preview also includes installing RKE2 on Windows clus
 
 Windows Support for RKE2 Custom Clusters requires choosing Calico as the CNI.
 
+>**Important:** Rancher will allow Windows workload pods to deploy on both Windows and Linux worker nodes by default. When creating mixed clusters in RKE2, you must edit the <b>NodeSelector</b> in the chart to direct the pods to be placed onto a compatible Windows node.
```
Comment (btat) [inline content/rancher/v2.6/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md:46]: Could you check if we have existing content on NodeSelector to point to in our docs? If not, adding a link to the upstream docs would be he good for users.

https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector

**Review by btat — CHANGES_REQUESTED** [review]

File: content/rancher/v2.6/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md (hunk 2 above)
Comment (jtravee) [reply content/rancher/v2.6/en/cluster-provisioning/rke-clusters/windows-clusters/_index.md]: Added upstream link as suggested.

**Review by rak-phillip — APPROVED** [review]
LGTM

**Review by btat — APPROVED** [review]

**Pull request merged**